packages/webapp/components/FooterNavBar.tsx

```diff
@@ -72,13 +72,10 @@ export default function FooterNavBar({
       className={classNames(
         'fixed !bottom-0 left-0 z-2 w-full',
         isNewMobileLayout
-          ? 'footer-navbar bg-gradient-to-t from-blur-baseline via-blur-bg via-70% to-transparent px-2 pt-2'
+          ? 'footer-navbar bg-gradient-to-t from-background-subtle via-background-subtle via-70% to-transparent px-2 pt-2'
```

Comment:
Should we also change the gradient stop positions? I remember they were defined differently in figma...

Reply:
The stop position was at a different number in Figma, but the issue is that the container in Figma does not compute for the whole container. I adjusted it a bit.

Reply:
And what is the point of starting with `background-subtle` color, then continuing with `background-subtle` color on the next stop? Can we just add `from-70%` instead?

Reply:
Right, yeah I think that's the better approach. Let me check the `from-70%`.